front-end/src/util/sorting.js

```diff
@@ -1,10 +1,22 @@
 export const default_sort = (data) => {
     return data
-        .sort((a, b) => {
-            return b['mentees'].length - a['mentees'].length})
+        .sort((a, b) => {return b['mentees'].length - a['mentees'].length})
```

Comment:
What is ´(a,b)´ meaning?

Reply:
When you pass a function to the `sort` method, it's used as a compare function which takes two arguments for deciding equality or precedence.

Reply:
@aclaudel remember that according to https://www.pivotaltracker.com/story/show/170422933 as an expected behavior the craftspeople with the same amount of mentees should be sorted alphabetically. 

I would like to pair in this, to check a little bit more of the React part